lib/node_modules/@stdlib/stats/base/variancech/lib/accessors.js

```diff
@@ -0,0 +1,94 @@
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MAIN //
+
+/**
+* Computes the variance of a strided array using a one-pass trial mean algorithm.
+*
+* ## Method
+*
+* -   This implementation uses a one-pass trial mean approach, as suggested by Chan et al (1983).
+*
+* ## References
+*
+* -   Neely, Peter M. 1966. "Comparison of Several Algorithms for Computation of Means, Standard Deviations and Correlation Coefficients." _Communications of the ACM_ 9 (7). Association for Computing Machinery: 496–99. doi:[10.1145/365719.365958](https://doi.org/10.1145/365719.365958).
+* -   Ling, Robert F. 1974. "Comparison of Several Algorithms for Computing Sample Means and Variances." _Journal of the American Statistical Association_ 69 (348). American Statistical Association, Taylor & Francis, Ltd.: 859–66. doi:[10.2307/2286154](https://doi.org/10.2307/2286154).
+* -   Chan, Tony F., Gene H. Golub, and Randall J. LeVeque. 1983. "Algorithms for Computing the Sample Variance: Analysis and Recommendations." _The American Statistician_ 37 (3). American Statistical Association, Taylor & Francis, Ltd.: 242–47. doi:[10.1080/00031305.1983.10483115](https://doi.org/10.1080/00031305.1983.10483115).
+* -   Schubert, Erich, and Michael Gertz. 2018. "Numerically Stable Parallel Computation of (Co-)Variance." In _Proceedings of the 30th International Conference on Scientific and Statistical Database Management_. New York, NY, USA: Association for Computing Machinery. doi:[10.1145/3221269.3223036](https://doi.org/10.1145/3221269.3223036).
+*
+* @param {PositiveInteger} N - number of indexed elements
+* @param {number} correction - degrees of freedom adjustment
+* @param {Object} x - input array object
+* @param {Function} x.get - accessor function
+* @param {integer} stride - stride length
+* @param {NonNegativeInteger} offset - starting index
+* @returns {number} variance
+*
+* @example
+* var x = {
+*     'get': function get( i ) {
+*         return [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0 ][ i ];
+*     }
+* };
+* var N = 4;
+* var stride = 2;
+* var offset = 1;
+*
+* var v = variancech( N, 1, x, stride, offset );
+* // returns 6.25
+*/
+function variancech( N, correction, x, stride, offset ) {
+    var mu;
+    var ix;
+    var M2;
+    var M;
+    var d;
+    var n;
+    var i;
+
+    n = N - correction;
+    if ( N <= 0 || n <= 0.0 ) {
+        return NaN;
+    }
+    if ( N === 1 || stride === 0 ) {
+        return 0.0;
+    }
+    ix = offset;
+
+    // Use an estimate for the mean:
+    mu = x.get( ix );
```

Comment:
This is not what we want. Follow the conventions in other packages already having accessor protocol support.